pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -156,8 +156,20 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
 				{
-					Verbs:     util.NewStringSet(authorizationapi.VerbAll),
-					Resources: util.NewStringSet(authorizationapi.ResourceAll),
+					// replicationControllerGetter
+					Verbs:     util.NewStringSet("get", "list"),
+					Resources: util.NewStringSet("replicationcontrollers"),
+				},
+				{
+					// RecreateDeploymentStrategy.replicationControllerClient
+					// RollingDeploymentStrategy.updaterClient
+					Verbs:     util.NewStringSet("get", "update"),
+					Resources: util.NewStringSet("replicationcontrollers"),
+				},
+				{
+					// RecreateDeploymentStrategy.hookExecutor
+					Verbs:     util.NewStringSet("get", "create"),
```

Comment:
Code that runs in the deployer pod, or in the deployment controller? Certainly possible I missed something. @ironcladlou, do you know if the deployer pod needs `list pods` permission?